src/main/resources/application.yaml

```diff
@@ -37,8 +37,12 @@ idam:
   redirectUrl: ${IA_IDAM_REDIRECT_URI:http://localhost:3002/oauth2/callback}
   s2s-auth:
     url: ${S2S_URL:http://service-auth-provider-api}
-    secret: ${S2S_SECRET_WORKFLOW_API:AAAAAAAAAAAAAAAC}
-    name: ${S2S_NAME_WORKFLOW_API:wa_workflow_api}
+    secret:
+      camunda: ${S2S_SECRET_TASK_CONFIGURATION_API:AAAAAAAAAAAAAAAC}
+      ccd: AAAAAAAAAAAAAAAC
```

Comment:
should make this an env var otherwise it won't work in aat :)

Reply:
replied on slack https://work-allocation.slack.com/archives/C016ELENM0S/p1604415532131200